### PR TITLE
Fixed a possible null pointer exception in BaseElement, as well as go…

### PR DESCRIPTION
**Pull request by fmcejudo**
…ing for more java 8 enhancement and supporting BaseElement with tests.

### CODE REVIEW

**Comment by erdemedeiros** [comment]
Hi @fmcejudo, thank you for your submission. I'm really sorry o we took that long to handle it. If you're still around could you, please, sign License/CLA and update the PR.

**Comment by fmcejudo** [comment]
Hi, still around! License signed. Thank for considering it.

**Comment by erdemedeiros** [comment]
Don't forget to rebase your pull request that's currently in conflict with the `develop` branch.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Hi @fmcejudo, please, do not touch to versions (files `VERSION` and `pom.xml`). You should be able to build your Pull Request without modifying those files. Let me know if you're encountering any problems related to that.
Header files should not be modified either: use the same existing header in the new files you add.

**Comment by fmcejudo** [comment]
sorry, my fork was not in sync with yours and I messed it up a bit. it is already solved

File: activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/BaseElement.java [hunk 1]
```diff
@@ -1,144 +1,135 @@
 /*
- * Copyright 2010-2020 Alfresco Software, Ltd.
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
```
Comment (erdemedeiros) [inline activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/BaseElement.java]: Please, revert this change in the header and update the header of `BaseElementTest` to match this one.

**Review by erdemedeiros — COMMENTED** [review]
Thank you for the updates. You only missed the headers of `BaseElement` and `BaseElementTest`. Then it should be ready to merge.

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Comment by erdemedeiros** [comment]
The CI is failing because it's detecting an extra white space at the end of the line `Copyright 2010-2020 Alfresco Software, Ltd. `. You need to fix it in `BaseElement` and `BaseElementTest`.

**Pull request merged**